chainer/datasets/sub_dataset.py

```diff
@@ -222,15 +234,18 @@ def get_cross_validation_datasets(dataset, n_fold, order=None):
     return splits
 
 
-def get_cross_validation_datasets_random(dataset, n_fold, seed=None):
+def get_cross_validation_datasets_random(dataset, n_folds, seed=None):
```

Comment:
```
chainer/datasets/sub_dataset.py:259:20: F821 undefined name 'kwargs'
chainer/datasets/sub_dataset.py:264:23: F821 undefined name 'kwargs'
```

Please add keyword argument `**kwargs` to `get_cross_validation_datasets_random`.